### ISSUE TITLE
[Refactor] Refactor Chip classes name

### ISSUE THREAD

**Issue by nouha06** [Orange-OpenSource/ouds-flutter#298]
In the core lib, internal chip component classes are currently prefixed with chip_. They should be renamed to use the ouds_chip_ prefix for consistency.

<img width="301" height="162" alt="Image" src="https://github.com/user-attachments/assets/88638667-bc77-42ea-8cc2-62901abd916f" />

**Comment by AhmedAmineZr** [comment]
@nouha06 fait attention y a aussi les button qui manque Ouds au debut , vérifie si y a pas d'autre composant qui ont besoin de la refacto aussi

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
